gpflow/features.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import tensorflow as tf
+from functools import singledispatch
+from abc import ABCMeta, abstractmethod
+
+from .param import Param, Parameterized
+from . import conditionals
+from ._settings import settings
+
+
+float_type = settings.dtypes.float_type
+
+
+class InducingFeature(Parameterized, metaclass=ABCMeta):
```

Comment:
This fails our python 2.7 tests...

Reply:
you can use `six`: `from six import with_metaclass` and change it to:
`class InducingFeature(with_metaclass(abc.ABCMeta, Parameterized)):`
to get Pytthon2/3 metaclasses

Reply:
Ok, bit ugly, but it'll have to do for now. At least it's only a one line change.